modular_skyrat/master_files/code/datums/storage/subtypes/pockets.dm

```diff
@@ -0,0 +1,24 @@
+
+// Shoe pockets
+/datum/storage/pockets/shoes/New()
+	. = ..()
+
+	add_holdable(list(
+	//Adds pistol magazines from guncargo companies in their respective order
+	/obj/item/ammo_box/magazine/multi_sprite/pdh,
+	/obj/item/ammo_box/magazine/multi_sprite/ladon,
+	/obj/item/ammo_box/magazine/multi_sprite/firefly,
+	/obj/item/ammo_box/magazine/multi_sprite/pdh_peacekeeper,
+	/obj/item/ammo_box/magazine/multi_sprite/mk58,
+
+	/obj/item/ammo_box/magazine/m45,
+	/obj/item/ammo_box/magazine/pepperball, //boot pepper
+
+	/obj/item/ammo_box/magazine/multi_sprite/g17,
+	/obj/item/ammo_box/magazine/multi_sprite/cfa_ruby,
+	/obj/item/ammo_box/magazine/multi_sprite/cfa_snub,
+
+	/obj/item/ammo_box/magazine/multi_sprite/makarov,
+
+	/obj/item/ammo_box/magazine/m9mm_aps
```

Comment:
Also, if it's just to add content to the `can_hold`, I'd honestly just say to add it directly to that variable in the /tg/ file, with Skyrat edit comments of course, for performance reasons. Mostly because it's initializing a lot of lists that won't be overly useful in the grand scheme of things, and adding some overall relatively unnecessary operations, when we could just have it be apart of the list it's meant to be apart of from the start.

Reply:
which review should i follow? do i format it correctly or do i make it nonmodular again, or do i keep the proc while making the listnonmoduar